cylc/flow/scripts/validate_install_play.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""cylc validate-install-play [OPTIONS] ARGS
+
+Validate install and play a single workflow.
+
+This script is equivalent to:
+
+    $ cylc validate /path/to/myworkflow
+    $ cylc install /path/to/myworkflow
+    $ cylc play myworkflow
+
+"""
+
+from ansimarkup import parse as cparse
+
+from cylc.flow import LOG
+from cylc.flow.scripts.validate import (
+    VALIDATE_OPTIONS,
+    wrapped_main as validate_main
+)
+from cylc.flow.scripts.install import (
+    INSTALL_OPTIONS, install, get_source_location
+)
+from cylc.flow.scheduler_cli import PLAY_OPTIONS
+from cylc.flow.option_parsers import (
+    CylcOptionParser as COP,
+    combine_options,
+    cleanup_sysargv,
+    ARGS, KWARGS
+)
+from cylc.flow.scheduler_cli import _play
+from cylc.flow.terminal import cli_function
+
+from typing import TYPE_CHECKING
+
+
+if TYPE_CHECKING:
+    from optparse import Values
+
+
+CYLC_ROSE_OPTIONS = COP.get_cylc_rose_options()
+VIP_OPTIONS = combine_options(
+    VALIDATE_OPTIONS, INSTALL_OPTIONS,
+    PLAY_OPTIONS, CYLC_ROSE_OPTIONS
+)
+
+
+def log_subcommand(command, ident):
+    """Log a command run as part of a sequence.
+
+    TODO: When implementing other Compound commands,
+    move this somewhere common.
+    """
+    print(cparse(
+        f'RUNNING: <b><cyan>$ cylc {command} {ident}</cyan></b>'))
```

Comment:
In the CLI context I think it's obvious what this means without the `RUNNING` prefix:

```suggestion
        f'<b><cyan>$ cylc {command} {ident}</cyan></b>'))
```